JavaScript/README.md

```diff
@@ -0,0 +1,86 @@
+## Getting Started
+
+Here's an example for creating a 128x128 array of OpenSimplex2 noise
+
+``` javascript
+// Create and configure FastNoise object
+let noise = new FastNoiseLite();
+noise.SetNoiseType(FastNoiseLite.NoiseType.OpenSimplex2);
+
+// Gather noise data
+let noiseData = [];
+
+for (let y= 0; y < 128; y++) {
+   for (let x= 0; x < 128; x++) {
+      if (typeof noiseData[x] == 'undefined']){
+         noiseData[x] = [];
+      }
+
+     noiseData[x][y] = noise.GetNoise(x,y);
+   }
+}
+
+// Do something with this data...
+```
+
+Method overloading explained.
+
+As JavaScript does not support method overloading i simulated it 
+inordered to make the user experence more continuitous between language versions.
+
+Here is a template for the method overloading:
+
+```javascript
+class FastNoise {
+   /**
+    * use if you want to have doc show up when called e.g public method
+    * @param {number} param1
+    * @param {number} param2
+    * @param {number} [param3]
+    */
+   methodWithDoc (param1,param2,param3/*optional*/) {
+      let R2 = (Param1,Param2) => {
+         //do something 2D
+      }
+
+      let R3 = (Param1,Param2,Param3) => {
+         //do something 3D
+      }
+
+
+      if (arguments.lenght === 2) {
```

Comment:
mild typo 😛 
(Also you might want to update this documentation entirely if you make the changes described in the code comments)